Updated the names of the CI tests and updated refine

elebitzero opened the pull request:
Fixes #5808

Changes proposed in this pull request:
- Updated the names of the CI e2e tests.
- Updated the refine script.

wetneb reviewed (approved):
We could have a documentation update too:
https://openrefine.org/docs/technical-reference/functional-tests#cicd

wetneb commented:
Updated the docs: https://github.com/OpenRefine/openrefine.org/commit/de1dfe51be6e8945f34f2fc747662b7229a32be3

elebitzero commented:
I updated again to change `e2e_test` to `e2e_tests` and removed the `chrome` parameter: https://github.com/OpenRefine/openrefine.org/commit/6c2ceea43dbc2f03921dda68c2cb4279f2fc7d5f
